### DISCUSSION TITLE
Is there a way to create a label with a clickable link?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [beeware/toga#2295] [Q&A]
something similar to the image below:
![image](https://github.com/beeware/toga/assets/98482469/e0bff8d7-5388-45e9-9fc2-70099dfcb696)

**Comment by freakboy3742** [comment]
Not at present. There's a proposal for a [LinkLabel](https://github.com/beeware/toga/issues/801), but there isn't even a draft implementation at this point.
